Automattic/wp-calypso
# replace (some) of $white sass vars with --color-white css custom prop

#### Changes proposed in this Pull Request

* Update all `$white` SASS variables to use `--color-white` custom prop (excluding usage in `rgba()` functions which I'll address in another PR and limiting to files in `client/`)

#### Testing instructions

* code: make sure all assignments are correct
* visually: smoke test calypso and see if there are any color regressions regarding white
